website/docs/developer/deployment-guide/deployment-upgrading-orca.md

```diff
@@ -75,7 +75,11 @@ To update your ORCA version:
 1.	Find the desired release on the [ORCA Releases page](https://github.com/nasa/cumulus-orca/releases).
 
 2.	Update the `source` in your Terraform deployment files for ORCA by
-replacing `vx.x.x` with the desired version of ORCA.
+replacing `vx.x.x` with the desired version of ORCA. If upgrading from `v8.x.x` to `v9.x.x` follow the below steps:
+    1. Run the Lambda deletion script found in `bin/delete_lambda.py` this will delete all of the ORCA Lambdas with a provided prefix. Or delete them manually in the AWS console.
+    2. Navigate to the AWS console and search for the Cumulus RDS security group.
+    3. Remove the inbound rule with the source of `PREFIX-vpc-ingress-all-egress` in Cumulus RDS security group.
```

Comment:
Any reason why we are removing this inbound rule?

Reply:
Yes so when you try and delete the `PREFIX-vpc-ingress-all-egress` it says it can't be deleted because it's associated with the Cumulus RDS security group as an inbound rule so in order to delete it we must remove the inbound rule. After users delete the `PREFIX-vpc-ingress-all-egress` security group and run the apply it will be re-associated with that Cumulus RDS group